packages/comet-uswds/src/hooks/use-file-input.tsx

```diff
@@ -0,0 +1,46 @@
+const useFileInput = () => {
+  const clear = (elemId: string): void => {
+    const elem = document.getElementById(elemId) as HTMLInputElement;
+    if (elem === null) {
+      return;
+    }
+
+    // Get the Parent element based on USWDS file input HTML structure
+    const parent = elem?.parentElement as HTMLFormElement;
```

Comment:
elem guranteed to be non-null 
` if (elem === null) {
      return;
    }` 
so optional chaining is redundant

Reply:
Removed. Good catch!